contrib/QAT-ZSTD-Plugin-cmake/CMakeLists.txt

```diff
@@ -0,0 +1,90 @@
+# Intel® QuickAssist Technology ZSTD Plugin (QAT ZSTD Plugin) is a plugin to Zstandard*(ZSTD*) for accelerating compression by QAT.
+# Out-of-tree means kernel don't have native support, user will build and install driver from external package: https://www.intel.com/content/www/us/en/download/765501.html
```

Comment:
Let's move the comment l. 2-4 to where the option is first used in l. 192 of contrib/CMakeLists.txt.